src/Compilers/VisualBasic/Test/Semantic/HasValidFonts.vb

```diff
@@ -3,7 +3,6 @@
 Imports System.Drawing
 Imports Roslyn.Test.Utilities
 
-
```

Comment:
Consider reverting as nothing else in this file is changed.

Reply:
Decided to keep as it's correct (no double blanks). I did edit this file but then reverted the change later. Kept this part.